ParkingLot/DataModels/ParkingBoyModels.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Linq;
+
+namespace ParkingLot.DataModels
+{
+    public class ParkingBoy
+    {
+        public ParkingBoy(Lot[] parkingLots, int id = 0)
+        {
+            this.Id = id;
+            this.ParkingLots = parkingLots;
+        }
+
+        public Lot[] ParkingLots { get; set; }
+        public int Id { get; }
+        public string TryParkToSpecificLot(Car car, Lot parkingLot, out Ticket ticket)
+        {
+            if (!parkingLot.IsAvailabe)
+            {
+                ticket = null;
+                return "Not enough positions.";
+            }
+
+            if (parkingLot.IsCarAlreadyHere(car.Plate))
+            {
+                ticket = null;
+                return $"Your car {car.Plate} is already in {parkingLot.LotName}.";
+            }
+
+            ticket = parkingLot.AcceptCar(car);
+            return string.Empty;
+        }
```

Comment:
It's great to have the try method. Since the return type may be different with the same input.